packages/sitecore-jss-angular/src/test-data/metadata-data.ts

```diff
@@ -0,0 +1,145 @@
+import { EditMode } from '@sitecore-jss/sitecore-jss/layout';
+
+export const layoutData = {
+  serverRoute: '/',
+  language: 'en',
+  sitecore: {
+    context: {
+      pageEditing: true,
+      editMode: EditMode.Metadata,
+      serverRoute: '/',
+      language: 'en',
+    },
+    route: {
+      name: 'main',
+      componentName: '',
+      fields: {
+        /* TODO: uncomment when doing Fields rework. Type mismatches otherwise
```

Comment:
We can uncomment right now, since this object doesn't have exact type. Also the shape that should be is:
```
text: {
  metadata: {
    fieldId: '123',
    fieldType: 'text',
    rawValue: 'textField',
  },
},
```
Apply the same change for the comments added below in this file